htdocs/comm/propal/class/propal.class.php

```diff
@@ -1713,6 +1714,13 @@ public function fetch_lines($only_product = 0)
 
 				$line->fetch_optionals();
 
+				// multilangs
+				if (! empty($conf->global->MAIN_MULTILANGS) && ! empty($objp->fk_product)) {
+        		$line=New Product($this->db);
```

Comment:
"New" must be "new" according to PHP syntax

Reply:
Also, because in most cases, we don't need the translations, to avoid to make too much request in a frequently called method, can you add a parameter $loadalsotranslation = 0 as a parameter of fetch_lines ?

Reply:
Also, enhancement must be pushed into develop branch now.
10.0 has been freeze several weeks ago.